src/Compilers/CSharp/Test/Semantic/Semantics/NativeIntegerTests.cs

```diff
@@ -7,6 +7,7 @@
 using System;
 using System.Collections.Immutable;
 using System.Linq;
+using System.Runtime.CompilerServices;
```

Comment:
>System.Runtime.CompilerServices [](http://example.com/codeflow?start=6&length=31)

Is this used?

Reply:
Interesting. This using wasn't used, but also wasn't grayed out in IDE.